mesheryctl/internal/cli/root/perf/view.go

```diff
@@ -0,0 +1,94 @@
+package perf
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+
+	"github.com/pkg/errors"
+	log "github.com/sirupsen/logrus"
+
+	"github.com/layer5io/meshery/mesheryctl/internal/cli/root/config"
+	"github.com/layer5io/meshery/mesheryctl/pkg/utils"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+var outFormatFlag string
+
+var viewCmd = &cobra.Command{
+	Use:     "view",
+	Short:   "view perf profile",
+	Long:    `See the configuration of your performance profile`,
+	Example: "mesheryctl perf view [ performance test profile name ]",
+	Args:    cobra.MaximumNArgs(1),
```

Comment:
```suggestion
	Args:    cobra.ExactArgs(1),
```
We always need the performance-id as an argument.